vizro-mcp/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM ghcr.io/astral-sh/uv:python3.12-bookworm-slim
+
+WORKDIR /app
+
+COPY pyproject.toml uv.lock ./
+RUN uv sync --frozen --no-install-project --no-dev --no-editable
+
+COPY . /app
+
+ENTRYPOINT ["uv", "run", "vizro-mcp"]
```

```suggestion
ENTRYPOINT ["uv", "run", "--no-sync", "vizro-mcp"]
```

I think? Since the dependencies are already up to date so we can skip this stage here. Please do check though.

---

we need another sync here so that the actual vizro-mcp is installed. otherwise there is error

```
╰─ docker run -i --rm --mount type=bind,src=<some path>,dst=<some path> vizro-mcp
error: Failed to spawn: `vizro-mcp`
  Caused by: No such file or directory (os error 2)
```